Packs/TOPdesk/ReleaseNotes/1_2_11.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### TOPdesk
+- Updated the Docker image to: *demisto/python3:3.10.13.89873*.
+- Add defaultCallerId
```

Comment:
Added a **Default Incident Caller ID**  parameter, which is use when creating new TOPdesk incidents without specifying a caller ID.